app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/yourinfo/YourInfoFragment.kt

```diff
@@ -31,44 +31,57 @@ class YourInfoFragment : Fragment(R.layout.contract_detail_your_info_fragment) {
                     (adapter as? YourInfoAdapter)?.submitList(
                         homeSection(
                             it.address.fragments.addressFragment,
-                            it.squareMeters,
-                            it.saType.displayName(requireContext())
-                        ) + coinsuredSection(it.numberCoInsured)
+                            it.saType.displayName(requireContext()),
+                            it.squareMeters
+                        ) + coinsuredSection(it.numberCoInsured) +
+                            YourInfoModel.Paragraph(
```

Comment:
This paragraph appears to be common to the `coinsuredSection`, could we add it to that function? That would reduce the repetition somewhat.